odroid-go-common/sdkconfig

```diff
@@ -20,7 +20,10 @@ CONFIG_LOG_BOOTLOADER_LEVEL_INFO=
 CONFIG_LOG_BOOTLOADER_LEVEL_DEBUG=
```

Comment:
This entire file needs to be omitted from the commit.

Reply:
Ah crap, I forgot about this. I'm not sure why the toolchain keeps wanting to rewrite it.

Reply:
I reverted it to your last commit.